README.md

```diff
@@ -78,19 +91,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 ```
 
-In the above example, we make the assumption that an user session is from when they open the app until the app enters the background.  How you define an user session is completely up to you.  It may be that you define it by the time they enter the application to 30 minutes.
+These lines initialize the amp and session instances that represent a single session in the Amp project corresponding to the `projectKey` that will be given to you. In the above example, we make the assumption that an user session is from when the app is created. How you define an user session is completely up to you.  It may be that you define it by the time they enter the application to 30 minutes.
 
 ### Observe
-Making `observe` requests can be called directly from your Amp instance with the Amp-iOS client.
 >Swift
 ``` Swift
-amp.observe("Signup", ["lang": "en", "timeOfDay": "evening"])
+amp.observe("CheckoutAmount", ["amount": amount])
 ```
 
 >Objective-C
 ```ObjectiveC
-[_amp observeWithName:@"Signup" properties:@{@"lang": @"en", @"timeOfDay": @"evening"}];
+[_amp observeWithName:@"CheckoutAmount" properties:@{@"amount": amount}];
 ```
+Most likely, you will want to include `observe` events to capture metrics (that measures how well your application is to your business goals) or the contexts that are relevant to them. You would place this within action or delegate methods to capture events or at the startup of your app to capture contexts which will help to improve upon your business goals.
```

Comment:
...events to capture `user context prior to decision-making and` capture metrics